src/usr/bin/google_set_hostname

```diff
@@ -61,7 +61,7 @@ if [ -n "$new_host_name" ] && [ "${new_host_name%%.*}" != "$old_hostname" ] && !
 
   hostnamectl=$(which hostnamectl 2> /dev/null)
   if [ -x "$hostnamectl" ]; then
-    hostnamectl hostname "${new_host_name%%.*}"
+    hostnamectl set-hostname "${new_host_name%%.*}"
```

Comment:
This command is deprecated. What distro are you seeing this on? If this is still necessary, prefer falling back to this command (`hostnamectl hostname [...] | hostnamectl set-hostname [...]`) rather than defaulting to it.

Reply:
We are seeing this on RHEL 8.10, Which would be a pretty recent kernel.